fuse/eval/metrics/libs/classification.py

```diff
@@ -61,6 +61,44 @@ def auc_roc(
             y_score=y_score, y_true=np.asarray(target) == pos_class_index, sample_weight=sample_weight, max_fpr=max_fpr
         )
 
+    @staticmethod
+    def auc_roc_mult_label(
+        pred: Sequence[Union[np.ndarray, float]],
+        target: Sequence[Union[np.ndarray, int]],
+        sample_weight: Optional[Sequence[Union[np.ndarray, float]]] = None,
+        average: str = "micro",
+        max_fpr: Optional[float] = None,
+    ) -> float:
+        """
+        Compute multi label auc roc (Receiver operating characteristic) score using sklearn
+        :param pred: prediction array per sample. Each element shape [num_classes]
```

Comment:
lets change num_classes -> num_labels

Reply:
it is still the number of classes. (num labels is the number of 1's In the vector)